xAPI.md

```diff
@@ -2643,23 +2643,52 @@ the owner of that extension IRI explicitly states that a particular form of code
 
 ## 1.0 Requests
 
-### 1.1 Request Types
+xAPI tracking is done via HTTP Requests from the Activity Provider (client) to the LRS (server).  This 
+specification offers guidance in some aspects of this communication.  Where no guidance is offered, it is 
+recommended that those implementing xAPI use current industry best practices.
+
+### 1.1 Request Methods
+
+xAPI supports the use of PUT, POST, GET, DELETE, and HEAD.  Use of other methods is beyond the scope of xAPI.  Each of the following sections details the use of each method and how xAPI may differ slightly.  For full 
+implementation details, see each of the [Resources](#resources) below.
 
 #### 1.1.1 HTTP PUT
 
+The HTTP PUT method is used to either create or update.  The enclosed entity is stored by the corresponding  
```

Comment:
I recommend deleting the "The enclosed..." sentence entirely and rewording the one after to be "If there is no resource at this location, it is created with the content of the request. If there is a resource, and the request meets any preconditions, the resource is replaced with the content of the request."